internal/pkg/service/buffer/store/configstore/configstore_test.go

```diff
@@ -66,22 +65,19 @@ func TestConfigStore_CreateReceiver(t *testing.T) {
 	err := store.CreateReceiver(ctx, config)
 	assert.NoError(t, err)
 
-	// Assert that it exists in the DB
-	encodedConfig, err := json.EncodeString(config, false)
-	assert.NoError(t, err)
-
-	r, err := d.etcdClient.KV.Get(ctx, "config", etcd.WithPrefix())
-	assert.NoError(t, err)
-
-	var found *string
-	for _, v := range r.Kvs {
-		if strings.HasPrefix(string(v.Key), ReceiverKey(config.ProjectID, config.ID)) {
-			temp := string(v.Value)
-			found = &temp
-			break
-		}
-	}
-	assert.Equal(t, *found, encodedConfig, "inserted config not found")
+	// Check keys
+	etcdhelper.AssertKVs(t, d.etcdClient, `
+<<<<<
+config/receiver/1000/github-pull-requests
+-----
+{
+  "receiverId": "github-pull-requests",
+  "projectId": 1000,
+  "name": "Github Pull Requests",
+  "secret": "%s"
+}
+>>>>>
+`)
```

Comment:
Doplnil som to do vsetkych testov, aj do `list*`, kde to byt asi nemusi.
Skontrolujte prosim tiez tie vystupy, ci tam nevidite nieco podozrive.